### PR TITLE
Makes girders melt

### PR DESCRIPTION
**Pull request by FreeStylaLT**
Having large plasma fires form neat outlines for each has-been room has never looked like it made sense, and it made rebuilding fired-up rooms surprisingly simple (in terms of rebuilding individual rooms), which seems to contradict the intention behind being able to melt _walls_ in the first place (thus, melting away the outline of a room and allowing the fire to spread).
🆑 FreeStylaLT
add:girders now melt in hot fires.
/🆑

### CODE REVIEW

**Comment by Crazylemon64** [comment]
I'd recommend you implement it using this instead of simply making them burn like a wooden barricade: [relevant proc to override](https://github.com/ParadiseSS13/Paradise/blob/master/code/LINDA/LINDA_fire.dm#L2-L3)

**Comment by Citinited** [comment]
Agree with crazylemon, they shouldn't burst into flame so much as just melt

**Comment by Plotron** [comment]
Changelog is missing.

File: code/game/objects/structures/girders.dm [hunk 1]
```diff
@@ -47,6 +47,11 @@
 		new /obj/item/stack/sheet/metal(get_turf(src))
 		qdel(src)
 
+/obj/structure/girder/temperature_expose(gas_mixture/air, exposed_temperature)
```
Comment (Crazylemon64) [inline code/game/objects/structures/girders.dm]: `datum/gas_mixture/air`

**Pull request merged**